connector/dao/src/main/java/it/pagopa/pdv/tokenizer/connector/dao/handler/ConnectorExceptionHandlingAspect.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.pdv.tokenizer.connector.dao.handler;
+
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughputExceededException;
+import it.pagopa.pdv.tokenizer.connector.exception.TooManyRequestsException;
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.annotation.AfterThrowing;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.stereotype.Component;
+@Slf4j
+@Aspect
+@Component
+public class ConnectorExceptionHandlingAspect {
+
+    @AfterThrowing(pointcut = "execution(* it.pagopa.pdv.tokenizer.connector.dao.*.*(..))", throwing = "ex")
```

Comment:
I suggest narrowing the focus of this advice to Connetor implementation(s) only, but i'm not sure the suggest code is correct.....give it a try 😉 
```suggestion
    @AfterThrowing(pointcut = "execution(public it.pagopa.pdv.tokenizer.connector.dao.*(..))", throwing = "ex")
```